### ISSUE TITLE
Transform unset on page resize

### ISSUE THREAD

**Issue by borovez** [tgomilar/paroller.js#33]
I am using the latest version of Paroller.js and jQuery.
I have no errors in console.

Paroller starts off good on page load, but as soon as I resize the page, the object I have floating, jumps down into its lowest position off of the screen. It doesn't maintain the objects current position.  So as soon as the window resizes, transform is set to "unset"... why is it doing this, it should be maintaining its current transform position.

As soon as I scroll, it jumps back up into position

**HTML:**
```
<img src="{{ asset('img/frontend/ipad.png') }}" width="1300px" class="fadeIn" data-wow-duration="1s" data-wow-delay="0.5s" class="mega my-element" data-paroller-factor="-0.2" data-paroller-factor-lg="-0.2" data-paroller-factor-md="-0.2" data-paroller-factor-sm="-0.2" data-paroller-factor-xs="-0.2" data-paroller-type="foreground" data-paroller-direction="vertical" style=" margin-left:-420px;   margin-top:200px;  transform: translateY(-262px);"/>
```

**SCRIPT:**
```
$(function(){
   $("[data-paroller-factor]").paroller();
});
```
VIDEO:
![2018-06-21_09-31-09](https://user-images.githubusercontent.com/1531030/41722262-ec202c96-7535-11e8-87e7-3cc904eaf1c0.gif)

**Comment by cadmicro** [comment]
Forgot to mention, this issue occurs on all browsers. Mozilla/Chrome/IE

**Comment by tgomilar** [comment]
Hi, fixed in [v1.4.1](https://github.com/tgomilar/paroller.js/releases/tag/1.4.1).
